src/main/webapp/app/exercises/shared/plagiarism/types/text/TextPlagiarismResult.ts

```diff
@@ -0,0 +1,19 @@
+import { TextComparison } from './TextComparison';
+import { PlagiarismResult } from '../PlagiarismResult';
+
+/**
+ * Result of the automatic plagiarism detection for text or programming exercises.
+ */
+export class TextPlagiarismResult extends PlagiarismResult {
+    comparisons: TextComparison[];
+
+    /**
+     * Duration of the plagiarism detection run in milliseconds.
+     */
+    duration: number;
+
+    /**
+     * Similarity distribution
```

Comment:
can you provide more details what you want to store here? what is meant with distribution?

Reply:
this might also be interesting for other types of exercises